packages/react/alert-dialog/src/AlertDialog.tsx

```diff
@@ -40,26 +40,26 @@ const [AlertDialogContext, useAlertDialogContext] = createContext<AlertDialogCon
   ROOT_NAME
 );
 
-const [AlertDialogContentContext, useAlertDialogContentContext] = createContext<
-  AlertDialogContentContextValue
->('AlertDialogContext', ROOT_NAME);
+const [
+  AlertDialogContentContext,
+  useAlertDialogContentContext,
+] = createContext<AlertDialogContentContextValue>('AlertDialogContext', ROOT_NAME);
 
 /* -------------------------------------------------------------------------------------------------
  * AlertDialog
  * -----------------------------------------------------------------------------------------------*/
 
 const AlertDialog: React.FC<React.ComponentProps<typeof Dialog>> = (props) => {
-  const { id: idProp, ...dialogProps } = props;
   const generatedId = makeId('alert-dialog', useId());
-  const alertDialogId = idProp || generatedId;
+  const alertDialogId = generatedId;
```

Comment:
I just removed that temporarily here because `id` was removed from `Dialog` but the changes to ids in `AlertDialog` will come in next PR.